lib/cdo/aws/s3.rb

```diff
@@ -87,6 +133,13 @@ def self.exists_in_bucket(bucket, key)
       create_client.head_object(bucket: bucket, key: key)
       return true
     rescue Aws::S3::Errors::NotFound, Aws::S3::Errors::Forbidden
+      if CDO.aws_s3_emulated?
```

Comment:
Should we log out something here?

Reply:
Oh, I could probably get rid of this, but this would, in theory, be used to redirect to the content on production to "backfill" the local buckets. But it is not implemented fully.